src/modules/Server/public/Get-SdnServerConfigurationState.ps1

```diff
@@ -53,7 +53,7 @@ function Get-SdnServerConfigurationState {
             }
         }
 
-        vfpctrl /list-vmswitch-port | Export-ObjectToFile -FilePath $OutputDirectory.FullName -Name 'vfpctrl_list-vmswitch-port' -FileType json
+        vfpctrl /list-vmswitch-port | Export-ObjectToFile -FilePath $OutputDirectory.FullName -Name 'vfpctrl_list-vmswitch-port' -FileType txt
```

Comment:
The same output as Get-SdnVfpVmSwitchPort except this is raw output of vfpctrl.

Reply:
Correct. I was debating about keeping both for now. For if any reason `Get-SdnVfpVmSwitchPort` was not able to parse the results with vfp and create a PSObject, I wanted to leverage the raw vfpctrl command to just dump the raw data. Since the files are small, did not see a large concern w/ keeping both formats.